GPU/D3D11/TextureCacheD3D11.cpp

```diff
@@ -477,11 +477,12 @@ class TextureShaderApplierD3D11 {
 			const float right = u2 * invHalfWidth - 1.0f + xoff;
 			const float top = v1 * invHalfHeight - 1.0f + yoff;
 			const float bottom = v2 * invHalfHeight - 1.0f + yoff;
-			// Points are: BL, BR, TR, TL.
-			verts_[0].pos = Pos(left, bottom, -1.0f);
-			verts_[1].pos = Pos(right, bottom, -1.0f);
-			verts_[2].pos = Pos(right, top, -1.0f);
-			verts_[3].pos = Pos(left, top, -1.0f);
+			float z = 0.0f;  // was -1.0f for some reason
```

Comment:
I think all that remains after this in D3D9 is implementing shader blending correctly, which is not done currently because there's no built-in variable in D3D9 pixel shaders to get the pixel position. We can work around that by passing it through from the vertex shader though.